std/algorithm/mutation.d

```diff
@@ -2586,7 +2586,7 @@ Params:
     i1 = first index
     i2 = second index
 */
-void swapAt(R)(auto ref R r, size_t i1, size_t i2)
+void swapAt(R)(ref R r, size_t i1, size_t i2)
```

Comment:
Right, the current code fails silently for some ranges. But, there are ranges which give its elements by reference that could be part of a UFCS chain, and therefore don't have a address. Those will no longer work with this change.